cylc/flow/main_loop/__init__.py

```diff
@@ -339,10 +339,10 @@ def load(config, additional_plugins=None):
         # load coroutines
         log = []
         for coro_name, coro in (
-                (coro_name, coro)
-                for coro_name, coro in getmembers(module)
-                if isfunction(coro)
-                if hasattr(coro, 'main_loop')
+                (corou_name, corou)
```

Comment:
I think the similarity of the two names could be tricky to read. Why not `coroutine_name` and `coroutine`. It's a mouthful, but only exists on four lines.

Also at the start of a loop like this it's a little like the written English language convention of "Three Letter Acronym [TLA]" Use TLA in rest of document".